### PR TITLE
Removed margin for CodeCatz image in navbar

### PR DESCRIPTION
**Pull request by anuschka**

### CODE REVIEW

**Comment by goranche** [comment]
why doesn't the CSS file have any indentation?

File: templates/base.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
 		<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
 		<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap-theme.min.css">
 		<link rel="stylesheet" href="{% static 'css/style.css' %}">
+		<link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
```
Comment (gandalfar) [inline templates/base.html]: This should come before our custom style.css. Move it one line up.

**Comment by anuschka** [comment]
Sorry. I messed it up. Will clean :smile:

File: templates/base.html (hunk 1 above)
Comment (anuschka) [inline templates/base.html]: Thanks!

**Comment by ialja** [comment]
There are still a few lines without indentation - check the Files changed tab :)

**Comment by anuschka** [comment]
Is this still not ok? Please hint :-)

**Comment by goranche** [comment]
we don't get a notification when a new commit is added to the PR, so I guess nobody noticed any change...

File: static/css/style.css [hunk 2]
```diff
@@ -45,16 +45,17 @@ body {
 .container .credit {
 	margin: 20px 0;
 }
-
```
Comment (goranche) [inline static/css/style.css]: Just out of curiosity, why did you remove this blank line, but (for example) added one on line 51?
it seems a bit inconsistent :wink:

**Comment by goranche** [comment]
@anuschka is this related to #70 ?

**Comment by anuschka** [comment]
Yes, you are right this is related to #20.

File: static/css/style.css (hunk 2 above)
Comment (anuschka) [inline static/css/style.css]: Well long story. At one point I did some changes in my style.css and someone else did on the main repository changes to the same line. Then I had to resolve the conflict in my local git repository and ended up removing with the -f option the style.css file in my local git. After that I cut and pasted the style css from the main repository and good old VIM did some rock and roll on my, yes you guessed indentation :smile:
